common/types/mapper/thrift/shared.go

```diff
@@ -3055,6 +3062,7 @@ func ToListWorkflowExecutionsRequest(t *shared.ListWorkflowExecutionsRequest) *t
 		PageSize:      t.GetPageSize(),
 		NextPageToken: t.NextPageToken,
 		Query:         t.GetQuery(),
+		IsCron:        t.IsCron,
```

Comment:
similar to IDL change, will impact how frontend is built.

Reply:
Frontend needs to set this field only when user sets it, frontend should work as is without making any changes for other cases.